database/factories/RoomFactory.php

```diff
@@ -27,6 +27,7 @@ public function definition()
             'floor' => $this->faker->RandomDigit,
             'building' => $this->faker->word,
             'status' => $this->faker->randomElement(["available", "unavailable"]),
+            'room_type' => $this->faker->randomElement(["lounge", "mezzanine","conference"]),
```

Comment:
Can we have these types defined centrally somewhere? Maybe as a `config` or as a static field on the Model?  @WindOfRussia is there a best practice for this?

Reply:
normally enums are used. but we're in php sooo the best next thing is to add static string variables to the Model.